.github/workflows/ci.yml

```diff
@@ -45,15 +45,34 @@ jobs:
           path: ~/.npm
           key: ${{ runner.os }}-test-unit-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
           restore-keys: ${{ runner.os }}-test-unit-${{ env.cache-name }}-
-      - run: npm ci
-      - run: npm run build:metadata
 
-      - name: Run Ganache
+      # Env var expansion workaround
+      # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
+      - name: Set ADDRESS_FILE
```

Comment:
let's remove the `ADDRESS_FILE` job as it's not needed. Most likely the DcokerHub login won't be needed either